src/DotNetWorker.Core/Context/FunctionContextLoggerExtensions.cs

```diff
@@ -31,7 +31,7 @@ public static ILogger GetLogger(this FunctionContext context, string categoryNam
         {
             return context.InstanceServices
                    .GetService<ILoggerFactory>()!
-                   .CreateLogger(categoryName);
+                   .CreateLogger(Logging.LogCategories.CreateFunctionUserCategory(categoryName));
```

Comment:
Just wanna understand why we're doing this:

`CreateFunctionUserCategory` -> The category for logs written from within user functions. Are all the logs here user logs? Curious why we need to wrap the categoryName this way

Reply:
This was a difference I spotted in the previous .NET Worker code and so made it match our previous logging behavior.

Reply:
Also yes. When you call `.GetLogger()` it's because you're the user and you're going to do some logging.